src/RobotInterface/include/BipedalLocomotion/RobotInterface/ISensorBridge.h

```diff
@@ -302,42 +302,90 @@ class ISensorBridge
+    {
+        return true;
+    };
 
     /**
-     * Helper method to maintain SensorLists struct by populating it from the configuration parameters
-     * @note the user may choose to use/not use this method depending on their requirements for the implementation
-     * if the user chooses to not use the method, the implementation must simply contain "return true;"
+     * Helper method to maintain SensorLists struct by populating it from the configuration
+     * parameters
+     * @note the user may choose to use/not use this method depending on their requirements for the
+     * implementation if the user chooses to not use the method, the implementation must simply
+     * contain "return true;"
      *
      * @param[in] handler  Parameters handler
      * @param[in] sensorBridgeOptions configured object of SensorBridgeOptions
      * @param[in] sensorLists SensorLists object holding list of connected sensor devices
      */
-    virtual bool populateSensorListsFromConfig(std::weak_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> handler,
-                                               const SensorBridgeOptions& sensorBridgeOptions,
-                                               SensorLists& sensorLists) { return true; };
+    virtual bool populateSensorListsFromConfig(
+        std::weak_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> handler,
+        const SensorBridgeOptions& sensorBridgeOptions,
+        SensorLists& sensorLists)
+    {
+        return true;
+    };
 
     /**
-     * Helper method to maintain SensorBridgeMetaData struct by populating it from the configuration parameters
-     * @note the user may choose to use/not use this method depending on their requirements for the implementation
-     * if the user chooses to not use the method, the implementation must simply contain "return true;"
+     * Helper method to maintain SensorBridgeMetaData struct by populating it from the configuration
+     * parameters
+     * @note the user may choose to use/not use this method depending on their requirements for the
+     * implementation if the user chooses to not use the method, the implementation must simply
+     * contain "return true;"
      *
      * @param[in] handler  Parameters handler
      * @param[in] sensorBridgeMetaData configured object of SensorBridgeMetadata
      * @param[in] sensorLists SensorLists object holding list of connected sensor devices
      */
-    virtual bool populateSensorBridgeMetaDataFromConfig(std::weak_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> handler,
-                                                        SensorBridgeMetaData& sensorBridgeMetaData) { return true; };
+    virtual bool populateSensorBridgeMetaDataFromConfig(
+        std::weak_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> handler,
+        SensorBridgeMetaData& sensorBridgeMetaData)
+    {
+        return true;
+    };
```

Comment:
Probably you can revert these changes, just to have the PR cleaner. In addition, the indentation is weird

Reply:
Done!